packages/grafana-ui/src/components/Button/Button.mdx

```diff
@@ -7,7 +7,7 @@ import { Button } from './Button';
 
 ## Primary
 
-Used for "call to action".
+Used for "call to action", i.e. triggering the main action. There should never be more than one on a page. If you need multiple buttons for different actions, decide which action is the most important and make that the primary `Button`. All other `Button` components should be secondary.
```

Comment:
There can be a case when there is no primary action, in such case one should use secondary for everything.

Reply:
Would it make sense to have it default to secondary, so you don't accidentally end up with 10 primary action buttons?